### PR TITLE
Chore: Update `CODEOWNERS` for edge to dataviz

### PR DESCRIPTION
**Pull request by nmarrs**
Update the codeowners file with rename of edge squad to dataviz

Will update ownership of other visualization related code from BI / user-essentials / etc. in a future PR

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -206,7 +206,7 @@ WORKFLOW.md @torkelo
 /devenv/docker/ha_test/ @grafana/backend-platform
 /devenv/docker/loadtest/ @grafana/backend-platform
 /devenv/docker/rpmtest/ @grafana/backend-platform
-/devenv/jsonnet/ @grafana/grafana-edge-squad
+/devenv/jsonnet/ @grafana/dataviz-squad
```
Comment (nmarrs) [inline .github/CODEOWNERS:209]: Not sure if dataviz should keep this vs. hand off to someone else (i.e. maybe to Todd in @grafana/multitenancy-squad)?

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -340,25 +340,25 @@ lerna.json @grafana/frontend-ops
...
+/public/app/core/components/Layers/ @grafana/dataviz-squad
 /public/app/features/all.ts @grafana/user-essentials
 /public/app/features/admin/ @grafana/grafana-authnz-team
 /public/app/features/annotations/ @grafana/user-essentials
 /public/app/features/api-keys/ @grafana/user-essentials
-/public/app/features/canvas/ @grafana/grafana-edge-squad
+/public/app/features/canvas/ @grafana/dataviz-squad
 /public/app/features/commandPalette/ @grafana/user-essentials
-/public/app/features/comments/ @grafana/grafana-edge-squad
+/public/app/features/comments/ @grafana/dataviz-squad
```
Comment (nmarrs) [inline .github/CODEOWNERS:350]: Same comment here - I think this is from an experimental feature Alexander worked on - so not sure we should keep this

File: .github/CODEOWNERS [hunk 3]
```diff
@@ -371,15 +371,15 @@ lerna.json @grafana/frontend-ops
 /public/app/features/profile/ @grafana/user-essentials
 /public/app/features/runtime/ @ryantxu
 /public/app/features/query/ @grafana/dashboards-squad
-/public/app/features/query-library/ @grafana/grafana-edge-squad
+/public/app/features/query-library/ @grafana/dataviz-squad
```
Comment (nmarrs) [inline .github/CODEOWNERS:374]: Not sure about keeping ownership on this as well

**Comment by nmarrs** [comment]
It appears that new @grafana/dataviz-squad will first need to be approved to be under the [Grafana Team](https://github.com/orgs/grafana/teams/grafana-team) so that it is recognized / has proper access

**Comment by tolzhabayev** [comment]
@nmarrs please rebase on the main to fix the codeowners validation error (it needs to include these changes https://github.com/grafana/grafana/pull/62689)
SCR-20230206-opl

**Review by tolzhabayev — CHANGES_REQUESTED** [review]
please rebase first

**Comment by nmarrs** [comment]
Closing as addressed by https://github.com/grafana/grafana/pull/63200

**Pull request closed without merging**